src/main/java/core/basesyntax/service/FileReaderImpl.java

```
@@ -0,0 +1,17 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        try {
+            return Files.readAllLines(Paths.get(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file: " + filePath);
```

This violates checklist item #6: "Don't ignore exceptions in try-catch blocks". When you catch an exception and re-throw it as a new one, you should include the original exception as the cause to preserve the full stack trace for debugging. You can do this by using the `RuntimeException(String message, Throwable cause)` constructor.